src/material/core/style/_vendor-prefixes.scss

```diff
@@ -18,10 +17,6 @@
   &::-webkit-input-placeholder {
     @content;
   }
-
-  &:-ms-input-placeholder {
```

Comment:
Somehow this change is causing a screenshot diff in Chrome, can we move it to a separate PR to investigate so we can merge the rest of this PR?

Reply:
Done.